### PR TITLE
Updated Circuit Colour System

### PR DESCRIPTION
**Pull request by melmdesign**
## New colour range introduces:
- A system which matches that used by SumUp's brand team
- A more accessible palette for UI design 
- The addition of purple and orange tones (to be used in the future for dataviz)

### CODE REVIEW

File: src/themes/circuit.js [hunk 1]
```diff
@@ -9,44 +9,71 @@ const neutrals = {
...
+  g900: '#356560'
 };
 
+const purples = {
+  p100: '#E9CFF2',
+  p200: '#D2A9D3',
+  p300: '#C781C9',
+  p500: '#B54DB3',
+  p700: '#8928A2',
+  p900: '#5F1D6B'
```
Comment (connor-baer) [inline src/themes/circuit.js]: The `p` namespace is already taken by the primary color palette.

File: src/themes/circuit.js [hunk 2]
```diff
@@ -58,7 +85,7 @@ const misc = {
   bodyColor: black,
   danger: reds.r500,
   success: greens.g700,
-  warning: yellows.y700
+  warning: yellows.y500
 };
 
 export const colors = {
```
Comment (connor-baer) [inline src/themes/circuit.js]: You need to add the new colors to this object to export them.

File: src/themes/circuit.js (hunk 1 above)
Comment (connor-baer) [reply src/themes/circuit.js]: How about violets & `v`?

**Comment by mlent** [comment]
@melmdesign Looks like the build is failing because of snapshot tests 🙄 

Can you run `yarn run test:unit:verbose -u` to update the snapshots? If you have any issues just let me know and I can come by and help.

**Comment by connor-baer** [comment]
I fixed the deployment in #366 and #368, rebased, and deployed to [circuit.sumup.com](https://circuit.sumup.com) for preview.

**Pull request merged**